mkosi

```diff
@@ -427,16 +434,21 @@ def install_fedora(args, workspace, run_build_script):
     if os.path.exists(gpg_key):
         gpg_key = "file://%s" % gpg_key
     else:
-        gpg_key = "https://getfedora.org/static/81B46521.txt"
+        gpg_key = "https://getfedora.org/static/{key}.txt".format(key=FEDORA_KEYS_MAP[args.release])
```

Comment:
If there's just one argument, using named keys seems overkill. I don't think this is an improvement.